### ISSUE TITLE
Bans dont kick players on other server

### ISSUE THREAD

**Issue by rourke750** [Civcraft/CBanManagement#18]

**Comment by suirad** [comment]
It should. Could you provide more information?
https://github.com/Civcraft/CBanManagement/blob/master/src/vg/civcraft/mc/cbanman/listeners/MercuryMessageListener.java#L46

**Comment by rourke750** [comment]
I believe he is talking about the /kick command.

**Comment by suirad** [comment]
Oh you want it to broadcast kicks?

**Comment by Maxopoly** [comment]
No, I think the issue was that if we banned someone on server X and he was online on server Y, he wouldnt get kicked and could stay online until he relogged.

**Comment by ttk2** [comment]
sounds like a strong argument for broadcasting kicks and bans.

**Comment by Maxopoly** [comment]
Broadcasting in the sense of /say ?

**Comment by ttk2** [comment]
no in the sense of mercury.

**Comment by Maxopoly** [comment]
Well it already does that, it's just not working for some reason
